docs/addon/_category_.json

```diff
@@ -0,0 +1,9 @@
+{
+  "label": "\uD83D\uDD0C Addon",
+  "position": 5,
+  "collapsed": true,
+  "link": {
+    "type": "generated-index",
+    "description": "Addons are additional useful packages that can be used in Fiber."
```

Comment:
This should say:

`Addon is an additional useful package that can be used in Fiber.`